index_v2.html

```diff
@@ -39,4 +39,4 @@ <h3>Sub Heading</h3>
 	</div>
 	
   </body>
-</html>
\ No newline at end of file
+</html>

```

Comment:
On my version2 I updates my meta data for Search Engine Results, Author, Keywords and Description so it relates to the Tawa College Education Outside The Classroom Website.